src/gentropy/datasource/gnomad/ld.py

```diff
@@ -448,3 +449,80 @@ def get_ld_matrix_slice(
                 .alias("r"),
             )
         )
+
+    @staticmethod
+    def get_locus_index(
+        session: Session,
+        study_locus_row: DataFrame,
+        ld_index_path: str,
+        window_size: int = 1_000_000,
+        major_population: str = "nfe",
+    ) -> DataFrame:
```

Comment:
Very-very theorietical problem I have with this function: this logic has nothing GnomAD specific in it. It uses a dataframe (`_matrix_index`) that we lift over from the original ld index. In this sense, I'm not sure if this is the right place for this function.